helm/logging-operator/values.schema.json

```diff
@@ -40,6 +40,9 @@
         "managementCluster": {
             "type": "object",
             "properties": {
+                "insecureCA": {
+                    "type": "boolean"
+                },
```

Comment:
Why that change ?

Reply:
It's not a change I introduced myself, it just appears that some previous changes to the values file were not forwarded to the values schema